cloud/packages/sdk/src/app/session/events.ts

```diff
@@ -128,6 +134,8 @@ export class EventManager {
   constructor(
     private subscribe: (type: ExtendedStreamType) => void,
     private unsubscribe: (type: ExtendedStreamType) => void,
+    private packageName: string,
+    private baseUrl?: string,
```

Comment:
maybe we shouldn't make baseUrl optional so we don't need to do:
`this.baseUrl || ""` in other spots in the code.